docs/servers/add-aws-server.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Add a Server to Amazon Web Services (AWS)
+intro: Devopness allows you to provision a server on AWS using Elastic Compute Cloud (EC2) and manage it through Devopness.
+links:
+    overview:
+    quickstart:
+    previous:
+    next:
+    guides:
+    related:
+    featured:
+---
+
+# Add a Server to Amazon Web Services (AWS)
+1. On Devopness, navigate to a project then select an environment
+1. If there's no other server in the selected environment, just click `ADD SERVER`
+1. If a list of resource cards is displayed, find the `Servers` card and click View on it
+1. On the upper-right corner of the list click `ADD SERVER`
+1. Choose `Amazon Web Services` as cloud provider
+1. Select an AWS credential using the drop-down menu
+    > If no credential is listed or you want to use a different one, please follow the [Add a Cloud Provider Credential](../cloud-provider-credentials/get-cloud-providers-credentails.md) guide
```

Comment:
This link `../cloud-provider-credentials/get-cloud-providers-credentails.md` is correct? I think it should link the page and not the Markdown file.

Reply:
Nice catch! 🔥